docs/quick-install.md

```diff
@@ -1,117 +1,90 @@
 <!--- OVERVIEW --->
 # Quick Install
 
+## Smart Agent Overview
 
-The SignalFx Smart Agent is a metric agent written in Go for monitoring infrastructure and application services in a variety of environments. It is a successor to our previous [collectd agent](https://github.com/signalfx/collectd), and still uses collectd internally on Linux; any existing Python or C-based collectd plugins will still work without modification. On Windows collectd is not included, but the agent can run python-based collectd plugins without collectd. C-based collectd plugins are not available on Windows.
+The SignalFx Smart Agent is a metric-based agent written in Go that is used to monitor infrastructure and application services from a variety of environments.
 
- - [Concepts](#concepts)
- - [Installation](#installation)
+The Smart Agent contains three main components:
 
+| Component | Description |
+|-----------|-------------|
+| Monitors  |  This component collects metrics from the host and applications. For a list of supported monitors and their configurations, see [Monitor Configuration](./monitor-config.md).            |
+| Observers |   This component collects metrics from services that are running in your environment. For a list of supported observers and their configurations, see [Observer Configuration](./observer-config.md).           |
+| Writer    |   This component collects metrics from configured monitors and then sends these metrics to SignalFx on a regular basis. If you are expecting your monitors to send large volumes of metrics through a single agent, then you must update the configurations. To learn more, see [Agent Configurations](./config-schema.md#writer).          |
 
-## Concepts
 
-The agent has three main components:
+## Review pre-installation requirements for the Smart Agent
 
-* _Monitors_ that collect metrics from the host and applications
+Before you download and install the Smart Agent on a **single** host, review the requirements below.
 
-* _Observers_ that discover applications and services running on the host
+(For other installation options, including bulk deployments, see [Advanced Installation Options](./advanced-install-options.md).)
 
-* a _Writer_ that sends the metrics collected by monitors to SignalFx
+| General requirements   |     Linux requirements      |  Windows requirements |
```

Comment:
I don't think a table format works here, because it's column 1 for everyone, then either column 2 or 3, which is a weird way for a table to work. 

I think 3 subheadings would work better.

Reply:
This has been updated.